airbyte-cdk/bulk/core/extract/src/main/kotlin/io/airbyte/cdk/operation/DiscoverOperation.kt

```diff
@@ -68,6 +67,7 @@ class DiscoverOperation(
         airbyteStream.withSourceDefinedPrimaryKey(
             if (isValidPK) discoveredStream.primaryKeyColumnIDs else listOf(),
         )
+        airbyteStream.isResumable = airbyteStream.sourceDefinedPrimaryKey.isNotEmpty()
```

Comment:
A pleasantly concise change!

Reply:
Also not one that will age particularly well; but I prefer to punt rather than anticipating future needs. Conceivably, a stream may not have a PK and yet still be resumable, and vice-versa.